### ISSUE TITLE
Solarized Grey Comment Highlighting

### ISSUE THREAD

**Issue by jojojames** [altercation/solarized#251]
Not sure if this is how Solarized is supposed to look in the terminal. The comments are grey while the MacVim version looks perfect.

Only setting on is the termtrans one, iterm2 palette is in solarized dark.

![Screen Shot 2013-03-24 at 9 54 44 PM](https://f.cloud.github.com/assets/1227856/296620/3c2017b0-9508-11e2-9b5b-86cf11483307.png)

**Comment by theNerd247** [comment]
+1 with this issue. I'm using urxvt on Arch Linux with the solarized color scheme for urxvt. 
I've got the following settings in my ~/.vimrc:

    let g:solarized_termcolors=256
    set background=dark
    colorscheme solarized

**Comment by TulkasLaugh** [comment]
I've had the same issue only on one of the Macs that I work on. I've tried all the possible solutions but you know what ended up working? Uninstalling then re-installing with Vundle. Go figure >.>

**Comment by ksalman** [comment]
@TulkasLaugh I recently re-installed Vundle - last week, and I didn't change anything for me >.<

**Comment by TulkasLaugh** [comment]
Yeah, I changed colorschemes and changed it back to solarized and it got messed up again. I gave up and switched to summerfruit256 for light scheme and vividchalk for dark scheme.

**Comment by albusshin** [comment]
I'm experiencing this issue as well.

**Comment by albusshin** [comment]
I've resolved this issue on my own machine, please see [this answer](https://stackoverflow.com/a/45870875/1831275)

**Comment by he44** [comment]
same issue here.
haven't found a solution yet

**Comment by he44** [comment]
Okay I think I solve the issue by using a different solarize.vim file. I named this file solarized1.vim and put it in my /.vim/colors, then i changed colorscheme in .vimrc. And all the gray highlights on comments disappeared...
The one I am using right now is from https://github.com/altercation/vim-colors-solarized. 

![screen shot 2017-12-12 at 2 28 04 pm](https://user-images.githubusercontent.com/15525366/33906968-30645bd0-df49-11e7-80ff-447c7a020959.png)

**Comment by hikmet-demir** [comment]
Any solutions
![image](https://user-images.githubusercontent.com/31572734/78347326-fb579d00-75a8-11ea-85b5-c1d6f73387ea.png)

My terminal looks like this empty places grey highlighted which is pretty annoying
